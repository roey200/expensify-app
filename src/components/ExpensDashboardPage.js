import React from 'react'
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'
import ExpensesSummary from './ExpensesSummary'

const ExpensDashboardPage = () => (
    <div>
        <ExpensesSummary />
        <ExpenseListFilters />
        <ExpenseList />
    </div>
)

export default ExpensDashboardPage