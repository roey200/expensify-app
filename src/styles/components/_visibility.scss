.show-for-mobile {
    @media (min-width: $desk-breakpoint) {
        display: none;
    }
}

.show-for-desktop {
    @media (max-width: $desk-breakpoint - .01rem) {
        display: none;
    } 
}