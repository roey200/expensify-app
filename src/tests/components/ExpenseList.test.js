import React from 'react'
import { shallow } from 'enzyme'
import { ExpenseList } from '../../components/ExpenseList'
import expenses from '../fixtures/expenses'

test('Should render ExpenseList with expense', () => {
    const wrapper = shallow(<ExpenseList expenses={expenses}/>)
    expect(wrapper).toMatchSnapshot()
})

test('Should render ExpenseList without expense', () => {
    const wrapper = shallow(<ExpenseList  expenses={[]}/>)
    expect(wrapper).toMatchSnapshot()
})