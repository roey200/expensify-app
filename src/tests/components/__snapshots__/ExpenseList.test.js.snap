// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseList with expense 1`] = `
<div>
  <ExpenseListItem
    amount={195}
    createdAt={0}
    description="Gum"
    id="1"
    key="1"
    note=""
  />
  <ExpenseListItem
    amount={109500}
    createdAt={-345600000}
    description="Rent"
    id="2"
    key="2"
    note=""
  />
  <ExpenseListItem
    amount={4500}
    createdAt={345600000}
    description="Credit Card"
    id="3"
    key="3"
    note=""
  />
  <h1>
    Expense List
  </h1>
</div>
`;

exports[`Should render ExpenseList without expense 1`] = `
<div>
  <p>
    No expenses
  </p>
  <h1>
    Expense List
  </h1>
</div>
`;
